ucp/_libs/ucx_api.pyx

```diff
@@ -226,7 +226,10 @@ cdef class UCXContext(UCXObject):
         # See <https://github.com/rapidsai/ucx-py/pull/377>
         ucp_params.features = (UCP_FEATURE_TAG |  # noqa
                                UCP_FEATURE_WAKEUP |  # noqa
-                               UCP_FEATURE_STREAM)
+                               UCP_FEATURE_STREAM |  # noqa
+                               UCP_FEATURE_RMA |  # noqa
+                               UCP_FEATURE_AMO32 |  # noqa
+                               UCP_FEATURE_AMO64)
```

Comment:
I think we should make the options optional. `__init__()` could accept a set of future constants and default to:
```python
def __init__(self, config_dict, ucp_features={UCP_FEATURE_TAG, UCP_FEATURE_WAKEUP):
```
or 

```python
def __init__(self, config_dict, ucp_features={"UCP_FEATURE_TAG", "UCP_FEATURE_WAKEUP"):
```

Reply:
I was actually thinking about a mechanism like that. How would the features be passed in? If the strings are used then that would involve a lookup table/dict() while the named version would involve the user doing something like features=[ucp.feature.wakup, ucp.feature.tag]. Which one would be prefered?

Reply:
I have no strong opinion on this but I prefer it as string with a lookup dict.